doc/source/multiscene.rst

```diff
@@ -13,6 +13,51 @@ examples will walk through some basic use cases of the MultiScene.
     These features are still early in development and may change overtime as
     more user feedback is received and more features added.
 
+MultiScene Creation
+-------------------
+There are two ways to create a ``MultiScene``. Either by manually creating and
+providing the scene objects,
+
+    >>> from satpy import Scene, MultiScene
+    >>> from glob import glob
+    >>> scenes = [
+    ...    Scene(reader='viirs_sdr', filenames=glob('/data/viirs/day_1/*t180*.h5')),
+    ...    Scene(reader='viirs_sdr', filenames=glob('/data/viirs/day_2/*t180*.h5'))
+    ... ]
+    >>> mscn = MultiScene(scenes)
+    >>> mscn.load(['I04'])
+
+or by using the :meth:`MultiScene.from_files <satpy.multiscene.MultiScene.from_files>`
+class method to create a ``MultiScene`` from a series of files. This uses the
+:func:`~satpy.readers.group_files` utility function to group files by start
+time.
```

Comment:
Could you add "by start time or other filenames parameters" since that can be controlled by the reader configuration (YAML) or the user by keyword argument? Just want to emphasize that it isn't just start time.

Reply:
Sure, added!